### ISSUE TITLE
Prevent empty issues, redirect issues that should go upstream

### ISSUE THREAD

**Issue by HonkingGoose** [testing-library/eslint-plugin-testing-library#431]
Once we're done with #430, we can think about forcing contributors to use the provided forms.

We can also configure the template chooser (screen you see when you click "new issue") to show a button with a link to upstream projects like ESLint with a custom text. This can help re-direct unwanted stuff to the proper place. 😉 

Example `config.yml`:

```yml
blank_issues_enabled: false
contact_links:
  - name: ESLint upstream
    url: https://github.com/eslint/eslint/issues
    about: Report problems with the ESLint program itself to the upstream project.
```

Read [GitHub Docs, configuring the template chooser](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) to learn more.

**Comment by Belco90** [comment]
Oh nice, I didn't know this was possible! I like this idea.
